src/main/kotlin/lottery/domain/lotto/Lotto.kt

```diff
@@ -0,0 +1,24 @@
+package lottery.domain.lotto
+
+class Lotto(
+    var numbers: List<Int> = emptyList()
```

Comment:
```suggestion
    val numbers: List<Int> = emptyList()
```
컬렉션은 외부에서 주솟값을 바꾸지 않아도 add, remove 함수를 통해 데이터를 조작할 수 있습니다.
컬렉션은 val을 활용해보면 어떨까요?